helm.tf

```diff
@@ -5,7 +5,7 @@ resource "helm_release" "this" {
   namespace        = var.namespace
   name             = var.helm_release_name
   version          = var.helm_chart_version
-  repository       = var.helm_repo_url
+  repository       = var.helm_repo_oci ? "oci://${local.helm_repo_url}" : "https://${local.helm_repo_url}"
```

Comment:
Breaking change; `var.helm_repo_url` will not be 1:1 with `repository`, because a protocol prefix will always be prepended. Should be documented.